cli-plugins/apps/reference/dependency-matrix.hbs.md

```diff
@@ -0,0 +1,18 @@
+# Dependency Matrix
+
+| TAP Version   | Apps CLI Version | Required Cartographer Version |
+| ------------- | -------------    | -------------                 |
+| TAP 1.1.x     | 0.5.x            | 0.3.x or higher               |
+| TAP 1.2.x     | 0.7.x            | 0.4.x or higher               |
```

Comment:
remove TAP 1.2 - support ended on April 31, 2023

Reply:
removed tap 1.1 and 1.2